src/components/webView/iframe.native.js

```diff
@@ -3,24 +3,16 @@ import { WebView } from 'react-native-webview'
 
 import useLoadingIndicator from '../../lib/hooks/useLoadingIndicator'
 
+const DOMReady = 'DOMContentLoaded'
 const DOMLoadedDispatcher = `(function () {
-    var messenger = window.ReactNativeWebView || parent;
+    var messenger = window.ReactNativeWebView;
 
-    if (!messenger || ('function' !== (typeof messenger.postMessage))) {
+    if ('undefined' === typeof messenger) {
       return;
     }
 
-    var documentUrl = location.href;
-    var DOMReady = 'DOMContentLoaded';
-
     var onDOMContentLoaded = function() {
-      var messagePayload = {
-        event: DOMReady,
-        target: 'iframe',
-        src: documentUrl
-      };
-
-      messenger.postMessage(messagePayload, '*');
+      messenger.postMessage('${DOMReady}');
```

Comment:
i dont understand why this was changed.
specifically removing "parent"

Reply:
it was totally wrong.. I've read docs for RNWebView
you have to send events through window.ReactNativeWebView only, and payload should be a string (e.g. JSON.stringify for objects).
I've tested it locally with native android app - it works only that way. An old implementation from iframe.native didn't worked